package/yast2-adsi.spec

```
@@ -12,26 +12,19 @@
 # license that conforms to the Open Source Definition (Version 1.9)
 # published by the Open Source Initiative.
 
-# Please submit bugfixes or comments via http://bugs.opensuse.org/
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
 
 Name:           yast2-adsi
-Version:        1.1
+Version:        1.2
 Release:        0
 Summary:        ADSI Edit for YaST
 License:        GPL-3.0
 Group:          Productivity/Networking/Samba
-Url:            http://www.github.com/yast-samba/yast-adsi
+Url:            https://www.github.com/yast/yast-adsi
```

Again, using https://github.com/yast/yast-adsi would be more consistent.
```suggestion
Url:            https://github.com/yast/yast-adsi
```